### PR TITLE
RHDEVDOCS-2866: Document ability to filter by the certification level…

### PR DESCRIPTION
**Pull request by HarshCasper**
JIRA ID: [RHDEVDOCS-2866](https://issues.redhat.com/browse/RHDEVDOCS-2866)

SME/QE Reviewer: @sanketpathak 

Peer Reviewer: @Preeticp 

Aligned Team: DevTools

This PR is for the OCP 4.8 Release.

### CODE REVIEW

File: modules/helm-filtering-helm-charts-by-certification-level.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="filtering-helm-charts-by-certification-level_{context}"]
+= Filtering Helm Charts by their certification level
+
+As a user, you can use the source filter to filter the Helm charts based on their certification level in the *Developer Catalog*.
+
+.Procedure
+
+. In the *Developer perspective*, navigate to the *+Add* view and select a project. From the *Developer Catalog*, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.
```
Comment (Preeticp) [inline modules/helm-filtering-helm-charts-by-certification-level.adoc:12]: This should be the second step - From the *Developer Catalog*, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.

File: modules/helm-filtering-helm-charts-by-certification-level.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="filtering-helm-charts-by-certification-level_{context}"]
+= Filtering Helm Charts by their certification level
+
+As a user, you can use the source filter to filter the Helm charts based on their certification level in the *Developer Catalog*.
```
Comment (Preeticp) [inline modules/helm-filtering-helm-charts-by-certification-level.adoc:8]: Replace with: You can filter Helm charts based on their certification level in the *Developer Catalog*.

File: modules/helm-filtering-helm-charts-by-certification-level.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+.Procedure
+
+. In the *Developer perspective*, navigate to the *+Add* view and select a project. From the *Developer Catalog*, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.
+
+. Click on the Source filter in the Helm catalog to filter the Helm charts based on provider type.
```
Comment (Preeticp) [inline modules/helm-filtering-helm-charts-by-certification-level.adoc:14]: Replace with: Use the filters to the left of the list of Helm charts to filter the required charts:
- Use the **Chart Repositories** filter to filter charts provided by **Red Hat Certification Charts** or **OpenShift Helm Charts**.
-  Use the **Source** filter to filter charts sourced from **Partners**, **Community** or **Red Hat**.
Certified charts are indicated with the (icon pic) icon.

File: modules/helm-filtering-helm-charts-by-certification-level.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+[NOTE]
+====
+Source filter will not be visible when there is only one provider type.
```
Comment (Preeticp) [inline modules/helm-filtering-helm-charts-by-certification-level.adoc:18]: The **Source**...

File: modules/helm-filtering-helm-charts-by-certification-level.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+====
+
+. Select a chart and read the description, README, and other details about the chart. A certified badge can also be viewed with the Helm chart title in the catalog page and details panel for some of the helm charts.
```
Comment (Preeticp) [inline modules/helm-filtering-helm-charts-by-certification-level.adoc:21]: Replace with: You can now select the required chart and install it. This should be the ending statement of the module, not a step.

**Review by Preeticp — COMMENTED** [review]
@HarshCasper I have left a few suggestions, PTAL.

File: modules/helm-filtering-helm-charts-by-certification-level.adoc (hunk 1 above)
Comment (HarshCasper) [reply modules/helm-filtering-helm-charts-by-certification-level.adoc:12]: Done!

File: modules/helm-filtering-helm-charts-by-certification-level.adoc (hunk 2 above)
Comment (HarshCasper) [reply modules/helm-filtering-helm-charts-by-certification-level.adoc:8]: Done!

File: modules/helm-filtering-helm-charts-by-certification-level.adoc (hunk 5 above)
Comment (HarshCasper) [reply modules/helm-filtering-helm-charts-by-certification-level.adoc:21]: Done!

File: modules/helm-filtering-helm-charts-by-certification-level.adoc (hunk 4 above)
Comment (HarshCasper) [reply modules/helm-filtering-helm-charts-by-certification-level.adoc:18]: Done!

File: modules/helm-filtering-helm-charts-by-certification-level.adoc (hunk 3 above)
Comment (HarshCasper) [reply modules/helm-filtering-helm-charts-by-certification-level.adoc:14]: Done!

File: modules/helm-filtering-helm-charts-by-certification-level.adoc [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+. In the *Developer perspective*, navigate to the *+Add* view and select a project.
+
+. From the *Developer Catalog* tile, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.
+
+. Use the filters to the left of the list of Helm charts to filter the required charts:
+* Use the *Chart Repositories* filter to filter charts provided by *Red Hat Certification Charts* or *OpenShift Helm Charts*.
+* Use the *Source* filter to filter charts sourced from *Partners*, *Community* or *Red Hat*. Certified charts are indicated with the icon.
+
+.Certified chart icon
```
Comment (Preeticp) [inline modules/helm-filtering-helm-charts-by-certification-level.adoc:20]: If we add the certified icon, then we can skip this screenshot.

Comment (HarshCasper) [reply modules/helm-filtering-helm-charts-by-certification-level.adoc]: Done!

**Pull request merged**